test/attr-for-selected.html

```diff
@@ -158,6 +168,21 @@
         testSelectItem(i);
       });
     });
+
+    suite('default attribute', function() {
+      var selector;
+      var items;
+
+      setup(function () {
+        selector = fixture('defaultAttribute');
+        items = Array.prototype.slice.apply(selector.querySelectorAll('div[some-attr]'));
+      });
+
+      test('setting non-existing value sets default', function() {
+        selector.select('non-existing-value');
+        assert.equal(selector.selectedItem, items[2]);
```

Comment:
Optional. You may or may not find this easier to read, but this could be written as:

```javascript
assert.equal(selector.selectedItem, Polymer.dom(selector).querySelector('[some-attr=default]');
```